src/components/project-card/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { StaticImageData } from "next/image";
+import "./index.css";
+
+interface ProjectCardProps {
+  title: string;
+  backgroundImage?: StaticImageData;
+  description1: string;
+  description2?: string;
+}
+
+const ProjectCard: React.FC<ProjectCardProps> = ({
+  description1,
+  backgroundImage,
+  description2,
+  title,
+}) => {
+  return (
+    <div className="project-card">
+      <div
+        className="project-image-container"
+        style={{ backgroundImage: `url(${backgroundImage?.src})` }}
```

Comment:
instead of using vanilla css background image, use an image tag from Next/Image, we'll be going more in-depth of this at tomorrow's meeting

We'll be passing in props via data files like this example here and then mapping through it but that's a later issue. here are some examples below
https://github.com/acm-ucr/witl-website/blob/dev/src/data/board.js  

ABOVE is where we have all the data such as images, etc. 

and we map through all our data within another file 

https://github.com/acm-ucr/witl-website/blob/dev/src/components/board/Board.jsx

but we only need to have the project component done, so something like this https://github.com/acm-ucr/witl-website/blob/dev/src/components/board/BoardCard.jsx (I know this is in js and not ts but this is just an example of how it will be done)